pkg/build/builder/dockerutil.go

```diff
@@ -124,6 +131,7 @@ func readInt64(filePath string) (int64, error) {
 		return -1, err
 	}
 	return val, nil
+
```

Comment:
nit: extra whitespace

Reply:
Funny there's an (very old) open issue for gofmt to address this particular case (https://github.com/golang/go/issues/6996)